packages/containers/src/ActionSplitDropdown/ActionSplitDropdown.connect.js

```diff
@@ -15,7 +15,10 @@ export function mapStateToProps(state, { actionId, actionIds } = {}) {
 		props = api.action.getActionInfo(context, actionId);
 	}
 	if (actionIds) {
-		props.items = actionIds.map(itemId => api.action.getActionInfo(context, itemId));
+		props.actionIds = actionIds;
```

Comment:
actionIds comes from external props, why do you want to add it from mapStateToProps result ?

Reply:
So it's to resolve it if the actionIds is in the first actionid resolution